packages/rich-text/README.md

```diff
@@ -6,9 +6,31 @@ npm install @contentful/field-editor-rich-text
 
 This package contains a React `RichTextEditor` component that is used as default for `RichText` field type in the Contentful web application.
 
-This editor app is built on top of [Slate](https://github.com/ianstormtaylor/slate), a third party framework for building highly customizable rich text editing solutions.
-
 ```js
 import 'codemirror/lib/codemirror.css';
 import { RichTextEditor } from '@contentful/field-editor-rich-text';
 ```
+
+## Migrating to v2
```

Comment:
A summary of the changes in v2. Let me know if I missed something. cc @DanweDE

Reply:
Sounds good!